src/main/webapp/3Dscript.js

```diff
@@ -0,0 +1,183 @@
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {OrbitControls} from 'https://threejs.org/examples/jsm/controls/OrbitControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// These are global objects.
+let scene;
+let camera;
+let renderer;
+let clock;
+let controls;
+let orientation;
+let trajectory;
+let dataLength;
+let pose;
+let poseRotation = {degrees: 0};
+let poseScalar = {multiplier: 1};
+let posePosition = {x: 0, z: 0};
+const apiKey = 'AIzaSyDCgKca9sLuoQ9xQDfHUvZf1_KAv06SoTU';
```

Comment:
would you have to hide this key somehow?

Reply:
I talked to Jared about this earlier, he said as long as we restrict the key to be used by just our domain we're okay to push the Maps API key.